docs/FRSKY_R9.md

```diff
@@ -85,14 +85,36 @@ Connections:
 - 5 Volts
 - GND
 
-Note: As the serial connection is inverted, this will have to be accounted for when connecting to the flight controller.  One solution is to use an inverter cable as mentioned above.  Alternatively, ArduPilot F7 and H7 based flight controllers have built-in inverters which can be enabled using with the SERIALx_OPTIONS parameter.
+Note: As the serial connection is inverted, this will have to be accounted for when connecting to the flight controller. One solution is to use an inverter cable as mentioned above. Alternatively, ArduPilot F7 and H7 based flight controllers have built-in inverters which can be enabled using with the SERIALx_OPTIONS parameter.
 
 JR Bay connector:
 - Rx, not used 
 - Tx, not used
 - Vin
 - GND
-- Pin 5 (S.Port) - Out (SBUS only)
+- Pin 5 (S.Port) - Out (SBus only)
+
+## R9M Lite Pro as Receiver ##
+
+The R9M Lite Pro transmitter module could be used as mLRS Tx module but it is not well suited for this. It can however be a great option as mLRS receiver, providing 1 W transmit power and hence excellent range in combination with a R9M Tx module.
+
+The R9M Lite Pro has no button; for binding the power cycle method is available (see [here](BINDING.md)). It also does not provide the CLI, and configuration hence must be done via the Lua script (see [here](LUA.md)).
```

Comment:
Perhaps:

'The R9M Lite Pro has no button therefore the power cycle method has to be used for binding (see [here](BINDING.md)).

Nitpick - should we mention the CLI at all?  Receivers by themselves don't use the CLI for configuration.

Reply:
1. gerat thx. Done
2. uuupppssss  ... that's not nitpick but pointing to a serious nonsense :) MANY THX. removed.